kata06/config/database.yml

```diff
@@ -17,6 +17,9 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  host: localhost
+  username: postgres
+  password: alceconbotas2
```

Comment:
It is not recommended to leave this configuration in Git, what you can do is add this file (database.yml) and exclude it in Git through the .gitignore, and later add a database.yml.example file with the access configuration example to the database.

This is recommended, to the different configurations that each user has, on their computer, of PostgreSQL.

Reply:
Ok, i will do the changes, thanks for the observations :D